domains/totocode.json

```diff
@@ -0,0 +1,12 @@
+{
+    "description": "Describe the use of this subdomain",
+    "repo": "https://github.com/TotoCodeFR/TotoCodeFR.github.io",
+    "owner": {
+        "username": "TotoCodeFR",
+        "email": "totocodefr@duck.com",
+        "twitter": "TotoCodeFR"
+    },
+    "record": {
+        "CNAME": "TotoCodeFR.github.io"
```

Comment:
Since the Github domain is redirecting to the is-a.dev one, I would like to see a preview of your website

Reply:
Should be fixed in a few minutes